docs/reference/db/migrations.md

```diff
@@ -65,4 +65,9 @@ _Example_
 
 ### Manually with the CLI
 
-See documentation about `db migrate` [command](./cli#migrate)
+See documentation about `db migrate` running `npx platformatic help db migrate`.
```

Comment:
Please link to: https://github.com/platformatic/platformatic/blob/main/docs/reference/cli.md#migrate

I think the original link should be: `../cli#migrate`